ssl/t1_enc.c

```diff
@@ -109,6 +109,7 @@ static int tls1_generate_key_block(SSL *s, unsigned char *km, size_t num)
   * record layer. If read_ahead is enabled, then this might be false and this
   * function will fail.
   */
+#ifndef OPENSSL_NO_KTLS_RX
```

Comment:
needs a space between the `#` and the `ifndef`.
likewise on the closing `#endif`

Reply:
fixed